chainsail_helpers/chainsail_helpers/pdf/__init__.py

```diff
@@ -35,3 +35,63 @@ def log_prob_gradient(self, x):
               log-probability gradient evaluated at x
         """
         pass
+
+
+class PosteriorPDF:
```

Comment:
Consider wrapping up abstract classes `PDF` and `PosteriorPDF` using Python's `abc` module (https://docs.python.org/3/library/abc.html#module-abc).
That way they errors may be cought by `mypy` or other typechecker if we run it and we get support in the IDE.

Reply:
:+1: addressed in https://github.com/tweag/chainsail-resources/pull/16/commits/531752eb15fbfeb595ce5b934e4d548a9c8d33f3.

Reply:
:+1: 
and a small comment in review round 2.